smt/surrogate_models/krg_based.py

```diff
@@ -1463,6 +1463,8 @@ def _predict_values(self, x: np.ndarray, is_acting=None) -> np.ndarray:
             X_cont = (X_cont - self.X_offset) / self.X_scale
 
         else:
+            n_eval, n_features_x = x.shape
+            _, ij = cross_distances(x, self.X_train)
```

Comment:
`ij` does not seem to used below and if `cross_distances` has no side-effect (what I hope!), you could remove this line.